hot/docker-host-stack.yaml

```diff
@@ -119,15 +119,11 @@ resources:
     type: "OS::Heat::CloudConfig"
     properties:
       cloud_config:
-        bootcmd:
-          - curl -L https://download.docker.com/linux/ubuntu/gpg -o /tmp/docker.gpg.asc
-          - gpg --dearmor -o /etc/apt/keyrings/docker.gpg /tmp/docker.gpg.asc
-          - chmod a+r /etc/apt/keyrings/docker.gpg
-        write_files:
-          - path: /etc/apt/sources.list.d/docker.list
-            permissions: '0644'
-            content: |
-              deb [arch=amd64 signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu jammy stable
+        apt:
+          sources:
+            docker.list:
+              source: deb [arch=amd64] https://download.docker.com/linux/ubuntu $RELEASE stable
```

Comment:
Huh, interesting! Didn't know about the option to refer to `$RELEASE` as a variable.

Reply:
That's pretty cool (and useful) indeed!